enterprise/app/root/root.tsx

```diff
@@ -164,6 +182,96 @@ export default class EnterpriseRootComponent extends React.Component {
             </OutlinedButton>
           </div>
         )}
+        {this.state.keyboardShortcutHelpShowing && (
```

Comment:
sorry I should've caught this in the last round - do you mind putting this in a separate package like `enterprise/app/shortcuts`? Then you could put it at the bottom along with the other modals like `<GroupSearchComponent>`, `<AlertComponent>` etc.

Reply:
Sure thing, done